simplyblock_core/env_var

```diff
@@ -1,6 +1,6 @@
-SIMPLY_BLOCK_COMMAND_NAME=sbcli-dev
-SIMPLY_BLOCK_VERSION=19.0.2
+SIMPLY_BLOCK_COMMAND_NAME=sbcli-mig
```

Comment:
updating this will fail the integration tests build. You could always install the sbcli from a specific branch using: 

```
pip install --upgrade git+https://github.com/simplyblock/sbcli.git@add_kubelet_restart_initContainer
```

and use sbcli-dev